src/UCommerce.SiteCore.Installer/Steps/DeleteRavenDB.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.IO;
+using Ucommerce.Installer;
+
+namespace Ucommerce.Sitecore.Installer.Steps
+{
+    public class DeleteRavenDB : AggregateStep
+    {
+        private readonly IInstallerLoggingService _loggingService;
+
+        public DeleteRavenDB(string appsPath, IInstallerLoggingService loggingService)
```

Comment:
Should appsPath not be a DirectoryInfo object to align with other places where we take these kinds of things? 

Maybe it could just take the SitecoreDirectory and take care of combining to the AppsDirectory.
See the EnableApp step if you need inspiration